test/specs/cartesian/YAxisSpec.js

```diff
@@ -194,4 +189,40 @@ describe('<YAxis />', () => {
     expect(wrapper.find('svg').children.length).to.equal(1);
     expect(wrapper.find('svg noscript').children.length).to.equal(1);
   });
+
+  it('Render identical ticks when data is hidden and includeHidden is true', () => {
```

Comment:
Using the ticks to validate this feature is fragile, because whenever the tick computation would change, this test would break. 

As far as I understand, the feature really is about the domain boundaries. Could you test whether the boundaries are identical? 

This might be possible getting the axis properties.

Reply:
I think this test should be resilient to tick computation changes, as it's not asserting on specific tick values, only that the values—whatever they are—stay the same when data items are hidden.

I was trying to keep this test as user-centric as possible as, to the user, static domain boundaries would manifest as consistent ticks on the yAxis, and consistent sizing of the graphical items. I've just added some assertions for bar heights to cover the latter. If you definitely think this is the wrong approach then I'm happy to change it, though I might need a little help as to which axis properties are best to test with!